stdlib/src/builtin/string.mojo

```diff
@@ -1446,6 +1446,48 @@ struct String(
 
         return output
 
+    fn splitlines(self, keepends: Bool = False) -> List[String]:
```

Comment:
I think it is better to move `keepends: Bool = False` to parameter. As following:

`fn splitlines[keepends: Bool = False](self) -> List[String]:`

This way the branch in the while loop will be backed at compile time:
```Mojo
@parameter
if keepends:
   output.append(self[current_offset : loc + 1])
else:
   output.append(self[current_offset:eol])
```

Reply:
Good point.